### DISCUSSION TITLE
Is there a way to uninstall Piku?

### DISCUSSION THREAD

**Discussion by Pipeliner** [piku/piku#324] [General]
I would like to know if there is a way to completely uninstall Piku from my server after trying it out. Is the process documented somewhere? It seems that bootstrap script only can install Piku.

**Comment by chr15m** [comment]
As stated in the bootstrap script:

```
    echo " ** WARNING **"
    echo " This script installs software and makes changes on this server."
    echo " Only use a freshly provisioned server which you do not mind"
    echo " being modified and reconfigured."
```

It's really not designed to be uninstalled and should be installed on a fresh VPS.

That said, removing the `piku` user and their home directory should uninstall most of it.

It also installs the following packages:

```
'bc', 'git', 'sudo', 'cron', 'build-essential', 'libpcre3-dev', 'zlib1g-dev', 'python3', 'python3-pip', 'python3-click', 'python3-dev', 'python3-virtualenv', 'python3-setuptools', 'nginx', 'acl', 'uwsgi-core', 'uwsgi-plugin-python3', 'nodeenv'
```

And some these files, depending on your configuration:

```
/usr/local/bin/uwsgi-piku
/etc/rc2.d/S01uwsgi-piku
/etc/systemd/system/uwsgi-piku.service
/etc/systemd/system/piku-nginx.path
/etc/systemd/system/piku-service.path
```

It also overwrites this file:
```
/etc/nginx/sites-available/default
```

I think that should be it. You an take a look at `playbooks/piku.yml` for more details.
